docarray/typing/tensor/abstract_tensor.py

```diff
@@ -225,6 +230,21 @@ def __docarray_validate_getitem__(cls, item: Any) -> Tuple[int]:
             raise TypeError(f'{item} is not a valid tensor shape.')
         return item
 
+    @classmethod
+    def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
+        field_schema.update(type='array', items={'type': 'number'})
+        if cls.__docarray_target_shape__ is not None:
+            shape_info = (
+                '[' + ', '.join([str(s) for s in cls.__docarray_target_shape__]) + ']'
+            )
+            if reduce(mul, cls.__docarray_target_shape__, 1) <= 256:
```

Comment:
make this 256 not a magic number

Reply:
I can pull it out as a constant, but it doesn't really come from anywhere honestly, so I think it'll have to stay hard coded